src/locales/en/translation.json

```diff
@@ -1287,10 +1287,13 @@
       "header": "You were not able to update your details with the GOV.UK ID check app",
       "content": {
         "paragraph1": "You can still use the service if your details are out of date.",
+        "paragraph1RepeatFraudCheck": "You will not be able to use your GOV.UK One Login to access the service.",
         "subHeading": "What would you like to do?",
         "formRadioButtons": {
           "continueOption": "Continue to the service without updating your details",
           "continueOptionHint": "Your proof of identity is valid even if your details have changed.",
+          "backOption": "Go back to the service you are trying to use",
```

Comment:
I think this should be "Go back to the service you _were_ trying to use" from the [designs](https://www.figma.com/design/nglBHtbzJYEeST43Iu1jW3/Identity-Pod---UCD-Hive?node-id=26170-21100&node-type=FRAME&t=REOnSSVibUkr7yf8-0)

Reply:
done